core/vm/evm.go

```diff
@@ -975,3 +979,19 @@ func (evm *EVM) CalculateMerkleRoot() (common.Hash, error) {
 	}
 	return combined.Hash(), nil
 }
+
+// Quorum
+//
+// validateMultitenancy - validates if multitenancy is authorized for this execution
+func validateMultitenancy(evm *EVM, code []byte, address common.Address) error {
```

Comment:
Not fan of `validateMultitenancy` naming. `IsMultitenancyEnabled` ?

Reply:
if I rename it to `IsMultitenancyEnabled`, should return a boolean and error? as naming `IsXXX` normally means that returns a boolean (but I need the `error` on this case to bubble up).

Reply:
I am still confused with those functions that do multiple stuff.

What I would expect is a sequence like

IsMultitenancyEnabled > if true, IsAuthorized() > if err, bubble up error

Reply:
But well maybe this is a more broader question not to be treated here and best to keep it this way.

Let me know your thoughts?